File: python/nim_array.py
nim = input("Masukkan NIM : ")

digit = [int(c) for c in nim]

total = 0
for d in digit:
    total +=d

maks = digit[0] 
for d in digit:
    if d > maks:
        maks = d

minim = digit[0]
for d in digit:
    if d < minim:
        minim = d

rata = total / len(digit)

rev = []
for i in range(len(digit)-1, -1, -1):
    rev.append(digit[i])

print("Digit        :", digit)
print("Total        :", total)
print("Maksimum     :", maks)
print("Minimum      :", min)
print("Rata-rata    :", rata)
print("Reverse array:", rev)

File: php/nim_array.php
<?php

$nim = readline("Masukkan NIM: ");

$digit = [];

for ($i = 0; $i < strlen($nim); $i++) {
    $digit[] = (int)$nim[$i];
}

$total = array_sum($digit);
$max = max($digit);
$min = min($digit);
$rata = $total / count($digit);

// Reverse array
$reverse = array_reverse($digit);

echo "\ndigit: ";
foreach ($reverse as $d) {
    echo $d . " ";
}

echo "\nTotal: $total";
echo "\nMaksimum: $max";
echo "\nMinimum: $min";
echo "\nRata-rata: $rata";

?>